src/apps/cs2cs.cpp

```diff
@@ -113,6 +114,16 @@ static void process(FILE *fid)
 
         z = strtod(s, &s);
 
+        /* To avoid breaking existing tests, we read what is a possible t    */
+        /* component of the input and rewind the s-pointer so that the final */
+        /* output has consistant behaviour, with or without t values.        */
+        /* This is a bit of a hack, 4D coordinates will be written to STDOUT */
+        /* but the output format speficied with -f is not respected for the  */
+        /* t component, rather it is forward verbatim from the input.        */
+        char *before_time = s;
+        time = strtod(s, &s);
```

Comment:
could be declared and initialized in the same line: double time = ...
Actually I'm wondering if time will not shadow the time() function.

Reply:
I've changed it, just in case